### PR TITLE
Rename classes containing Akka to Pekko

### PR DESCRIPTION
**Pull request by mdedetrich**
Renames all Scala/Java classes that happen to contain `Akka` to instead have `Pekko`

Resolves: #66

### CODE REVIEW

**Comment by mdedetrich** [comment]
> Could you update https://github.com/apache/incubator-pekko/issues/66 with the fully qualified names of the classes that changed name? Useful for writing a migration script later.

Will do

File: akka-actor-tests/src/test/scala/org/apache/pekko/PekkoVersionSpec.scala [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      intercept[UnsupportedPekkoVersion] {
+        PekkoVersion.require("PekkoVersionSpec", "2.5.6", "2.5.5")
+      }
+    }
+
+    "succeed if current Akka version is SNAPSHOT" in {
+      PekkoVersion.require("PekkoVersionSpec", "2.5.6", "2.5-SNAPSHOT")
+    }
+
+    "succeed if current Akka version is timestamped SNAPSHOT" in {
```
Comment (jxnu-liguobin) [inline akka-actor-tests/src/test/scala/org/apache/pekko/PekkoVersionSpec.scala:67]: Pekko version?

Comment (mdedetrich) [reply akka-actor-tests/src/test/scala/org/apache/pekko/PekkoVersionSpec.scala]: Thanks

File: akka-actor/src/main/scala/org/apache/pekko/PekkoException.scala [hunk 2]
```diff
@@ -29,6 +29,6 @@ trait OnlyCauseStackTrace { self: Throwable =>
 /**
  * This exception is thrown when Akka detects a problem with the provided configuration
```
Comment (jxnu-liguobin) [inline akka-actor/src/main/scala/org/apache/pekko/PekkoException.scala:30]:
```suggestion
 * This exception is thrown when Pekko detects a problem with the provided configuration
```

File: akka-actor/src/main/scala/org/apache/pekko/dispatch/ForkJoinExecutorConfigurator.scala [hunk 3]
```diff
@@ -13,7 +13,7 @@ object ForkJoinExecutorConfigurator {
   /**
    * INTERNAL AKKA USAGE ONLY
```
Comment (jxnu-liguobin) [inline akka-actor/src/main/scala/org/apache/pekko/dispatch/ForkJoinExecutorConfigurator.scala:14]:
```suggestion
   * INTERNAL PEKKO USAGE ONLY
```

Comment (jxnu-liguobin) [inline akka-actor/src/main/scala/org/apache/pekko/dispatch/ForkJoinExecutorConfigurator.scala:14]: There are many `INTERNAL AKKA USAGE ONLY` comments

File: akka-serialization-jackson/src/main/scala/org/apache/pekko/serialization/jackson/JacksonObjectMapperProvider.scala [hunk 4]
```diff
@@ -271,10 +271,10 @@ object JacksonObjectMapperProvider extends ExtensionId[JacksonObjectMapperProvid
 
   private def isModuleEnabled(fqcn: String, dynamicAccess: DynamicAccess): Boolean =
     fqcn match {
-      case "org.apache.pekko.serialization.jackson.AkkaTypedJacksonModule" =>
+      case "org.apache.pekko.serialization.jackson.PekkoTypedJacksonModule" =>
         // akka-actor-typed dependency is "provided" and may not be included
```
Comment (jxnu-liguobin) [inline akka-serialization-jackson/src/main/scala/org/apache/pekko/serialization/jackson/JacksonObjectMapperProvider.scala:275]:
```suggestion
        // pekko-actor-typed dependency is "provided" and may not be included
```

**Comment by mdedetrich** [comment]
@jxnu-liguobin So comments/documentation that just has the name Akka will be renamed in another PR (I like to keep things separate incase there are problems). This PR is strictly just about renaming classes that contain the name `Akka` and any comments that are **directly** affected (i.e. if a comment is referring to `SomeAkkaClass`).

**Comment by mdedetrich** [comment]
Going to merge the PR, updates to the Akka name in general will be done in a future PR.

**Pull request merged**